examples/fineFoods/admin/mui/src/pages/couriers/list.tsx

```diff
@@ -0,0 +1,137 @@
+                ),
+                flex: 1,
+            },
+            {
+                field: "gsm",
+                headerName: t("couriers.fields.gsm"),
+                flex: 1,
+            },
+            {
+                field: "email",
+                headerName: t("couriers.fields.email"),
+                valueGetter: ({ row }) => row.email,
+                flex: 1,
+            },
+            {
+                field: "comment",
+                headerName: t("couriers.fields.address"),
+                // eslint-disable-next-line react/display-name
+                renderCell: ({ row }) => (
+                    <Tooltip title={row.address}>
+                        <Typography
+                            sx={{
+                                textOverflow: "ellipsis",
+                                overflow: "hidden",
+                            }}
+                        >
+                            {console.log("row", row)}
+                            {row.address}
+                        </Typography>
+                    </Tooltip>
+                ),
+                flex: 1,
+            },
+            {
+                field: "actions",
+                headerName: t("table.actions"),
+                type: "actions",
+                getActions: ({ row }) => [
+                    <GridActionsCellItem
+                        key={1}
+                        label={t("buttons.edit")}
+                        icon={<Edit color="success" />}
+                        onClick={() => edit("couriers", row.id)}
+                        showInMenu
+                    />,
+                    <GridActionsCellItem
+                        key={2}
+                        label={t("buttons.delete")}
+                        icon={<Close color="error" />}
+                        onClick={() => {
+                            mutateDelete({
+                                resource: "couriers",
+                                id: row.id,
+                                mutationMode: "undoable",
+                            });
+                        }}
+                        showInMenu
+                    />,
+                ],
+            },
+        ],
+        [],
+    );
+
+    const { dataGridProps } = useDataGrid<ICourier>({
+        columns,
+        permanentFilter: [
+            {
+                field: "status",
+                operator: "eq",
+                value: "pending",
+            },
+        ],
+    });
+
+    return (
+        <List>
+            <DataGrid
+                {...dataGridProps}
+                autoHeight
+                density="comfortable"
+                rowsPerPageOptions={[10]}
+                sx={{
+                    "& .MuiDataGrid-cell:hover": {
+                        cursor: "pointer",
+                    },
+                }}
+                onCellClick={(row) => {
+                    show("couriers", row.id);
+                }}
```

Comment:
```suggestion
                 onRowClick={({ id }) => {
                      show("orders", id);
                  }}
```
I think, you should use `onRowClick` instead of `onCellClick`.